checks.d/network_ext.py

```diff
@@ -0,0 +1,21 @@
+"""
+Collects extended network metrics.
+"""
+# stdlib
+import re
+
+# project
+from checks import AgentCheck
+from utils import network
+
+
+class NetworkExt(AgentCheck):
```

Comment:
We have this as a separate check for now, but I think this would actually belong in the network.py check, possibly with a flag to enable extended metrics.